pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -159,6 +159,13 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 				fmt.Fprintln(o.Out)
 			}
 
+		// path "/" we tried to ping is forbidden, but the server is still reachable
+		case clientcmd.IsForbidden(result.Error()):
```

Comment:
@liggitt not that I know. If we decide to ping something else then this will likely not be needed anymore.

@smarterclayton Yeah, it's been here forever but I agree there are much better options. Maybe `/oapi` or `/oauth`?